packages/app/ui/modal/ModalHeader.tsx

```diff
@@ -1,13 +1,24 @@
 "use client";
 
-import { Dialog } from "@headlessui/react";
 import { PropsWithChildren } from "react";
 
-export function ModalHeader({ children }: PropsWithChildren) {
+import { cx } from "class-variance-authority";
+import { Dialog } from "@headlessui/react";
+
+interface ModalHeaderProps extends PropsWithChildren {
+  withDivider?: boolean;
+}
+
+export function ModalHeader({
+  children,
+  withDivider = false,
+}: ModalHeaderProps) {
   return (
     <Dialog.Title
       as="div"
-      className="flex items-center justify-between w-full border-b border-surface-50 px-4 py-3"
+      className={cx("flex items-center justify-between w-full px-4 py-3", {
+        "border-b border-surface-50": withDivider,
+      })}
     >
```

Comment:
Are we defining some `ModalHeader` without divider? If not, I don't see why we should define this

Reply:
Will remove it, it was intended to be used for the Toast component and enhance future modals that may use a divider between the content and the footer, but now is not needed as the Toast component was created with another approach.